node/network/availability-distribution/src/requester/fetch_task/mod.rs

```diff
@@ -326,6 +326,17 @@ impl RunningTask {
 		&mut self,
 		validator: &AuthorityDiscoveryId,
 	) -> std::result::Result<ChunkFetchingResponse, TaskError> {
+		gum::debug!(
```

Comment:
I think this is too noisy for debug. There are lots of chunks requested, trace would be better here.